apps/judicial-system/backend/migrations/20240503141513-update-case.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      Promise.all([
```

Comment:
Nitpick, but consider removing `Promise.all` because it's not needed for only `add/removeColumn`